### DISCUSSION TITLE
Allow using constants on the left side of a filter condition

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12722] [Feature Requests]
I have added a field `access_level` in `directus_roles` to get more granularity than App/Admin access to hide/show fields.
The following works
```
{
	"_and": [
		{
			"id": {
				"_lt": "$CURRENT_ROLE.access_level"
			}
		}
	]
}
``` 
_It is just a syntax validation as it is irrelevant to compare the `id` of the item to the `access_level`._

Why can't we have?
```
{
	"_and": [
		{
			"3": {
				"_gt": "$CURRENT_ROLE.access_level"
			}
		}
	]
}
``` 
It shows `3 _gt $CURRENT_ROLE.access_level` instead of `3 Greater Than $CURRENT_ROLE.access_level` and it is not working anyway.

Is there another way to show/hide fields based on the $CURRENT_ROLE.access_level?

**Comment by rijkvanzanten** [comment]
You can't do that, as we simply haven't implemented support for it 😅  We'll treat this as the feature request for this functionality 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
